helix-term/src/application.rs

```diff
@@ -316,7 +316,12 @@ impl Application {
                     }),
             );
         }
+
         self.config.store(Arc::new(config));
+
+        // Update all the relevant members in the editor after replacing
+        // the configuration.
+        self.editor.refresh_config();
```

Comment:
It might be better to call `editor.refresh_config()` in the `handle_config_events()` function since that will update the editor specific config on both `Refresh` and `Update` config events. The `Update` event is triggered on a runtime `:set` invocation and `Refresh` is triggered on `:config-reload`, so as of now auto-pairs is not configurable using `:set` (the PR fixes it for `:config-reload` only).

Reply:
Makes sense, I'll look into that. Thanks